README.md

```diff
@@ -4002,6 +4002,35 @@ _You can enable the following settings in Xcode by running [this script](resourc
   ```
 
   </details>
+  
+* <a id='remove-empty-extensions'></a>(<a href='#remove-empty-extensions'>link</a>) **Remove empty, non-conforming, extensions.** [![SwiftFormat: emptyExtension](https://img.shields.io/badge/SwiftFormat-emptyExtension-008489.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#emptyExtension)
+
+  <details>
+
+  #### Why?
+  Improves readability since the code has no effect and should be removed for clarity.
+  
+  ```swift
+  // WRONG
+  extension String {}
+  
+  extension [Foo: Bar] {}
+  
+  extension Array where Element: Foo {}
+  
+  extension String: Equatable {}
+  
+  @GenerateBoilerPlate
+  extension Foo {}
+
+  // RIGHT
+  extension String: Equatable {}
+  
+  @GenerateBoilerPlate
+  extension Foo {}
```

Comment:
I think we can simplify the example in the style guide -- the examples here don't need to be exhaustive as the test cases in SwiftFormat. Maybe we also use a type that fits the space theme of other examples.

```suggestion
  // WRONG: The first extension is empty and redundant.
  extension Planet {}
  extension Planet: Equatable {}
  
  // RIGHT: Empty extensions that add a protocol conformance aren't redundant.
  extension Planet: Equatable {}
```

Reply:
Updated!